tardis/atomic.py

```diff
@@ -19,15 +19,13 @@ class AtomDataNotPreparedError(Exception):
 
 logger = logging.getLogger(__name__)
 
-default_atom_h5_path = os.path.join(os.path.dirname(__file__), 'data', 'atom_data.h5')
 
 def data_path(fname):
-    data_dir = os.path.join(os.path.dirname(__file__), 'data')
-    return os.path.join(data_dir, fname)
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data', fname)
```

Comment:
I have a built-in PEP8 checker in Pycharm. I had compromised line wrap to 120 characters, but I switched it to 80 characters just now. I removed PEP8 errors from the part of code in this PR. Rest will be "_edited on the fly_" :wink: